65black/StackOverflow-lite
# #160140943 Ensure User Permissions When Editing Answers

#### What does this PR do?
A user can currently edit an answer that they did not originally post. This PR aims to fix that.

#### Description of Task to be completed?
Fixes bug on the following endpoint:

`POST api/v1/questions/:questionId/answers/:answerId`

#### How should this be manually tested?
After cloning the repo and installing all dependencies with `npm install`

- Create a new user and post an answer to a question
- Create another user and send a PUT request to the answer made by the first user
- Add this to your request body: _content:_ <your_new_answer> and send it.
- You should get an error message and your request will fail.

#### What are the relevant pivotal tracker stories?
#160140943

#### Screenshots (if appropriate)
![error](https://user-images.githubusercontent.com/36507972/44796566-1f6d2400-aba5-11e8-9781-2aa0c31f800e.PNG)
